openbb_terminal/cryptocurrency/defi/coindix_model.py

```diff
@@ -4,15 +4,19 @@
 import logging
 from typing import Optional
 
+import urllib3
 import pandas as pd
 import requests
 
 from openbb_terminal.decorators import log_start_end
 from openbb_terminal.helper_funcs import get_user_agent
 
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
To disable Insecure Request Warning I disabled it on coindix_model level, but maybe it should be moved to some more flags/config related place ?

Reply:
For now I think this looks good yea!